src/Entities/Pickup.cpp

```diff
@@ -92,12 +96,14 @@ class cPickupCombiningCallback :
 
 
 
-cPickup::cPickup(double a_PosX, double a_PosY, double a_PosZ, const cItem & a_Item, bool IsPlayerCreated, float a_SpeedX /* = 0.f */, float a_SpeedY /* = 0.f */, float a_SpeedZ /* = 0.f */)
+cPickup::cPickup(double a_PosX, double a_PosY, double a_PosZ, const cItem & a_Item, bool IsPlayerCreated, float a_SpeedX /* = 0.f */, float a_SpeedY /* = 0.f */, float a_SpeedZ /* = 0.f */, int a_LifetimeTicks /* = 6000 = 5 minutes */, bool a_CanCombine /* = true */)
```

Comment:
I think having the defaults here, although commented, is not practical. They serve no purpose here and can change.